library/library_example.cc

```diff
@@ -1,62 +1,67 @@
 #include <stdio.h>
-#include "../vowpalwabbit/parser.h"
-#include "../vowpalwabbit/vw.h"
+#include <cmath>
+#include "vowpalwabbit/parser.h"
+#include "vowpalwabbit/vw.h"
 
 using namespace std;
 
 
-inline feature vw_feature_from_string(vw& v, string fstr, unsigned long seed, float val)
+inline feature vw_feature_from_string(vw &v, string fstr, unsigned long seed, float val)
```

Comment:
Stylistically, vw& v makes more sense to me than vw &v since vw& is a type while v is a name.  This is obviously not superimportant.